packages/server/src/utilities/rowProcessor/bbReferenceProcessor.ts

```diff
@@ -39,3 +39,33 @@ export async function processInputBBReferences(
 
   return result.join(",")
 }
+
+export async function processOutputBBReferences(
+  value: string,
+  subtype: FieldSubtype
+) {
+  if (typeof value !== "string") {
+    return value
+  }
+
+  const result = []
+
+  switch (subtype) {
+    case FieldSubtype.USER:
+      for (const id of value.split(",").filter(x => !!x)) {
```

Comment:
Each BB reference type should be a CSV of just different IDs ideally, so we could probably move this to top level in the function, just a `const validIds = value.split(",").filter(id => !!id)`

I assume ideally as well this will use the new user cache function to fetch the users from a user ID array?

Reply:
Yup, I am changing the cache usage in another branch once merged to develop